common/metrics/metric_defs.go

```diff
@@ -420,6 +420,10 @@ const (
 	HistoryClientGetDLQReplicationMessagesScope = "HistoryClientGetDLQReplicationMessages"
 	// HistoryClientGetShardScope tracks RPC calls to history service
 	HistoryClientGetShardScope = "HistoryClientGetShard"
+	// HistoryClientIsActivityTaskValidScope tracks RPC calls to history service
+	HistoryClientIsActivityTaskValidScope = "HistoryClientIsActivityTaskValid"
+	// HistoryClientIsWorkflowTaskValidScope tracks RPC calls to history service
+	HistoryClientIsWorkflowTaskValidScope = "HistoryClientIsWorkflowTaskValidS"
```

Comment:
Could we provide some more meaningful comments here?

Reply:
no, all these comments should be deleted, they have no value. (and the constants should be deleted too and embedded in generated code (which is almost all of where they're used))